core/src/test/scala/za/co/absa/spline/harvester/plugin/embedded/SnowflakePluginSpec.scala

```diff
@@ -0,0 +1,58 @@
+package za.co.absa.spline.harvester.plugin.embedded
+
+import org.apache.spark.sql.{SaveMode, SparkSession}
+import org.apache.spark.sql.execution.datasources.SaveIntoDataSourceCommand
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.mockito.MockitoSugar
+import za.co.absa.spline.harvester.plugin.Plugin.WriteNodeInfo
+import za.co.absa.spline.harvester.builder.SourceIdentifier
+import org.mockito.Mockito._
+
+class SnowflakePluginSpec extends AnyFlatSpec with Matchers with MockitoSugar {
```

Comment:
Is there any possibility to add end-to-end integration tests to test compatibility with the real Spark?
Like, for example a Snowflake docker container or maybe official mock/test library?